test/production/app-dir/turbopack-build-marker/turbopack-build-marker.test.ts

```diff
@@ -0,0 +1,13 @@
+import { nextTestSetup } from 'e2e-utils'
+;(process.env.TURBOPACK ? describe : describe.skip)(
+  'turbopack-build-marker',
+  () => {
+    const { next } = nextTestSetup({
+      files: __dirname,
+    })
+
+    it('should have Turbopack build marker', async () => {
+      expect(await next.hasFile('.next/IS_TURBOPACK_BUILD')).toBe(true)
```

Comment:
```suggestion
      expect(await next.hasFile('.next/IS_TURBOPACK_BUILD')).toBe(!!process.env.TURBOPACK)
```

you could run the test for webpack too